README.md

```diff
@@ -9,7 +9,8 @@ added to it over time.
 
 ## Technical Requirements
 
-OS X 10.9 and above.
```

Comment:
I would suggest to also remove line 7-8 "It's still in heavy development" :-)

Reply:
Good catch- thanks!

Reply:
Done